docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md

```diff
@@ -0,0 +1,256 @@
+    | **DataVolume Name**                         | The name of your `DataVolume`.                                                                                                                                                                    |                                   |
+    | **Overhead Percentage for DataVolume Size** | Set an overhead percentage for your `DataVolume` compared to the OVA specification. This parameter is optional and can be skipped with the value `-1`.                                            |                                   |
+    | **Access Mode for the PVC**                 | Set the access mode for your `DataVolume`. Ensure that your configured CSI supports your selection.                                                                                               | `ReadWriteMany` / `ReadWriteOnce` |
+    | **Create a PVC with VolumeMode=Block**      | Indicate whether to set `Block` volume mode on the `DataVolume`.                                                                                                                                  | `y` / `N`                         |
+    | **StorageClass**                            | The storage class on the destination that will be used to create the VM volume.                                                                                                                   |                                   |
+    | **CDI upload proxy URL**                    | Optionally, provide a URL to upload the CDI custom resource.                                                                                                                                      |                                   |
+
+11. The import may take a few minutes to complete. The Palette CLI outputs the path for your OVA configuration file.
+    Make a note of it.
+
+12. Your imported OVA is now ready to be deployed to a new VM inside your VMO cluster. Execute the following command to
+    start an interactive shell and begin the deployment process. Replace the path placeholder with the path of your OVA
+    configuration file.
+
+    ```shell
+    palette vmo deploy-ova --config-file <path-to-ova-configuration-file>
+    ```
+
+    The Palette CLI prompts you for information regarding the VM you want to deploy.
+
+    | **Parameter**               | **Description**                                   | **Values** |
+    | --------------------------- | ------------------------------------------------- | ---------- |
+    | **VM Namespace**            | The namespace to create your VM into.             |            |
+    | **VM Name**                 | The namespace of your VM.                         |            |
+    | **Start VM Automatically?** | Indicate whether to start your VM after creation. | `y`/ `N`   |
+
+13. Depending on the configuration of your OVA, you may be prompted to provide value. The deployment of your VM should
+    complete within a few minutes.
+
```

Comment:
We may want to call out the fact that the proposed VM spec will be opened in a text editor and that at a minimum, you must delete the first two lines of the file.

Info:
> Minor edits to the auto-generated VM spec may be required; e.g., you may need to specify Multus VLAN configuration for each network interface, as the CLI does not yet support eliciting this information during the import process.